akvo/rsr/models/organisation.py

```diff
@@ -278,13 +277,7 @@ def knowledge(self):
         def all_projects(self):
             "returns a queryset with all projects that has self as any kind of partner"
             from .project import Project
-            return Project.objects.filter(partnerships__organisation__in=self).distinct()
-
-        def public_projects(self):
-            """
-            Returns a queryset with all public projects that has self as any kind of partner.
-            """
-            return self.all_projects().public()
+            return Project.objects.of_partner(self).distinct()
```

Comment:
Ah, sorry. I get the question, now! Yes, they are essentially doing almost the same thing. Except, passing a single organisation to `of_partners` won't work, since it expects an iterable. 

```python
org = Organisation.objects.get(id=1)
Project.objects.all().of_partners(org)  # will fail
Project.objects.all().of_partners([org]) # works
Project.objects.all().of_partner(org) # same as above
```